apps/src/templates/sectionProgress/SectionProgressToggle.jsx

```diff
@@ -38,4 +44,10 @@ class SectionProgrsesToggle extends React.Component {
   }
 }
 
-export default SectionProgrsesToggle;
+export const UnconnectedSectionProgrsesToggle = SectionProgrsesToggle;
+
+export default connect(state => ({}), dispatch => ({
+  setCurrentView(viewType) {
+    dispatch(setCurrentView(viewType));
+  },
+}))(SectionProgrsesToggle);
```

Comment:
Uh... what's with the typo in this name throughout the file?

Reply:
oops!